src/concepts/essentials/httpql.md

```diff
@@ -2,99 +2,167 @@
 
-- **req**: For HTTP requests.
-- **resp**: For HTTP responses.
-- **preset**: For Filter Presets.
+::: info
 
-The **Preset** Namespace is a bit different, it doesn't have a `Field` nor an `Operator`. View the [Filters](/reference/features/overview/filters.md) documentation for more information.
+- The `preset` and `source` Namespaces do not have a `Field` or an `Operator`. View [preset values](#preset)/[source values](#source) for usage.
+- The `source` Namespace is available in [Search](/reference/features/logging/search.md).
+:::
 
-## 2. Field
+## Field
 
-The second part of Filter Clause is the `Field`. Fields differ based on the `Namespace`.
-We will add more Fields eventually. [Let us know](https://github.com/caido/caido/issues/new?template=feature.md&title=New%20HttpQL%20field:) which ones you need!
+The **Fields** that Caido supports include:
 
 ### req
 
-- **ext**: The file extension (if we detected one). Extensions in Caido always contain the leading `.` (like `.js`). We are liberal in what we accept as an extension.
-- **host**: The hostname of the target server.
-- **method**: The HTTP Method used for the request in uppercase. If the request is malformed, this will contain the bytes read until the first space.
-- **path**: The path of the query, including the extension.
-- **port**: The port of the target server.
-- **raw**: The full raw data of the request. This allows you to search on things we currently don't index (like headers).
+- `ext`: The file extension (_if present_). Extensions in Caido always contain the leading `.` (_such as `.js`_).
+- `host`: The hostname of the target server.
+- `method`: The HTTP Method used for the request in uppercase. If the request is malformed, this will contain the bytes read until the first whitespace.
+- `path`: The path of the query, including the extension.
+- `port`: The port of the target server.
+- `raw`: The full raw data of the request. This allows you to search by elements that Caido currently does not index (_such as headers_).
+- `created_at`: The date and time the request was sent.
+
+::: tip
+Caido is liberal in what is accepted as an extension.
+:::
 
 ### resp
 
-- **code**: Status code of the reponse. If the response is malformed, this will contain everything after `HTTP/1.1` and the following space.
+- `code`: The status code of the reponse. If the response is malformed, this will contain everything after `HTTP/1.1` and the following whitespace.
+- `raw`: The full raw data of the response. This allows you to search by elements that Caido currently does not index (_such as headers_).
+- `roundtrip`: The total time taken for the request to travel from the client to the server and for the response to travel back from the server to the client.
 
-- **raw**: The full raw data of the response. This allows you to search on things we currently don't index (like headers).
+::: info
+View [roundtrip values](#roundtrip) for supported time formats.
```

Comment:
Roundtrip is always an integer in milliseconds, the time formats are for the `created_at` field.